unit-test/background/tracker-allowlist-tests.js

```diff
@@ -1,10 +1,30 @@
+const tdsStorageStub = require('./../helpers/tds.es6')
+const tds = require('../../shared/js/background/trackers.es6')
+const tdsStorage = require('../../shared/js/background/storage/tds.es6')
+
 const refTests = require('./reference-tests/tracker-radar-tests/TR-domain-matching/tracker_allowlist_matching_tests.json')
 const allowList = require('./../..//shared/js/background/allowlisted-trackers.es6')
+const config = require('./../data/extension-config.json')
+
+describe('Tracker allowlist tests:', () => {
+    beforeAll(() => {
+        config.features.trackerAllowlist = {
+            state: 'enabled',
+            settings: {
+                allowlistedTrackers: require('./../background/reference-tests/tracker-radar-tests/TR-domain-matching/tracker_allowlist_reference.json')
+            }
+        }
+
+        tdsStorageStub.stub({ config })
+
+        return tdsStorage.getLists()
+            .then(lists => tds.setLists(lists))
+    })
 
-describe('Tracker allowlist  tests:', () => {
     refTests.forEach(test => {
-        const result = allowList(test.site, test.request)
```

Comment:
having something happen outside of `it` means that it runs before `beforeAll` which is a bug that I chased for like an hour 😰